libs/test/Test/Golden.idr

```diff
@@ -257,6 +269,10 @@ runTest opts testPath = forkIO $ do
     printTiming True  clock msg = putStrLn (unwords [msg, show clock])
     printTiming False _     msg = putStrLn msg
 
+    wrapColor : Int -> Bool -> String -> String
+    wrapColor color bold s = "\x1b[" ++ (if bold then "1;" else "0;") ++
+      show color ++ "m" ++ s ++ "\x1b[0m"
+
```

Comment:
Can this (and its callers) be replaced by using `Libraries.Control.ANSI`?

Reply:
Hi @andrevidela, I think that since this is a very lightweight use ( only color / bold or not bold) this 3 line function would be better than importing something more feature-rich. If more color/effects/modes are desired in the feature it could be better to use Control.ANSI

Reply:
I agree, however, contrib is already a dependency of the `test` framework (https://github.com/idris-lang/Idris2/blob/f7f944a2367ebb3c0c882ad30d4f9ea687ea051c/libs/test/test.ipkg) and adding `import Control.ANSI` doesn't seem to have as much of a cost as re-inventing the wheel here.

There is also a benefit in terms of readability `show $ bolden $ colored Green "success"` reads much better than `wrapColor 32 False "success"`

Reply:
I am having issues mixing up bolden and colored together since

```
bolden : String -> DecoratedString
colored : Color -> String -> DecoratedString
```

And the only `DecoratedString -> other` is a field accesor or `show = escapeSGR dstr.sgr ++ dstr.str ++ escapeSGR [Reset]`

Will use Control.ANSI without bold for the time being; pushed the changes.

Reply:
I didn't realise that, maybe that's also the opportunity to revise the `ANSI` api. I think @ShinKage was the one to write it. What do you think @ShinKage ? should we open an issue to update the ANSI API to allow colored and bolded text?

Reply:
Well, not gonna bikeshed the use of bold so this could be merged with only the color changes.

Reply:
Yes

Reply:
We can open a PR to improve the ANSI API and add the change there.
On the matter of the API, you can already have bolded and colored text, but you have to construct or unwrap `DecoratedString` manually. We can probably add a function that merges the decoration annotations of multiple helpers like bolden or colored, just for ergonomics.

Reply:
Merginig would be great, yeah I could have done that wraping/unwrapping but then the 2 line wrapColor helper would have ended shorter than using Control.ANSI. unu